test/test_mypy.py

```diff
@@ -0,0 +1,52 @@
+import tempfile
+import unittest
+
+from mypy import api
+
+from test.config import PYPROJECT_TOML
+
+
+class TestMyMypyPlugin(unittest.TestCase):
+    def test_plugin_no_issue(self):
+        test_code = """
+import luigi
+import gokart
+
+
+class MyTask(gokart.TaskOnKart):
+    # NOTE: mypy shows attr-defined error for the following lines, so we need to ignore it.
+    # TODO: remove the following comments after fixing the issue.
+    foo: int = luigi.IntParameter() # type: ignore
+    bar: str = luigi.Parameter() # type: ignore
+
+MyTask(foo=1, bar='bar')
+        """
+
+        with tempfile.NamedTemporaryFile(suffix='.py') as test_file:
+            test_file.write(test_code.encode('utf-8'))
+            test_file.flush()
+            result = api.run(['--config-file', str(PYPROJECT_TOML), test_file.name])
+            self.assertIn('Success: no issues found', result[0])
+
+    def test_plugin_invalid_arg(self):
+        test_code = """
+import luigi
+import gokart
+
+
+class MyTask(gokart.TaskOnKart):
+    # NOTE: mypy shows attr-defined error for the following lines, so we need to ignore it.
+    # TODO: remove the following comments after fixing the issue.
+    foo: int = luigi.IntParameter() # type: ignore
+    bar: str = luigi.Parameter() # type: ignore
+
+MyTask(foo='1') # foo is int, but it is str and bar is missing.
+        """
+
+        with tempfile.NamedTemporaryFile(suffix='.py') as test_file:
+            test_file.write(test_code.encode('utf-8'))
+            test_file.flush()
+            result = api.run(['--config-file', str(PYPROJECT_TOML), test_file.name])
+            self.assertIn('error: Missing named argument "bar" for "MyTask"  [call-arg]', result[0])
```

Comment:
Is it possible to ignore missing named argument error? In some cases, we set the parameter with config file.

https://gokart.readthedocs.io/en/latest/setting_task_parameters.html#set-parameter-at-config-file

Reply:
Ok, I'll check this

Reply:
made this optional.
We can make the parameters optional or required based on reading yaml, which is future work.

[08e2b74](https://github.com/m3dev/gokart/pull/384/commits/08e2b74a4b96dec5a2f5c268962b405a2ce54b18)